academic_observatory_workflows/database/schema/doi/doi_2021-11-25.json

```diff
@@ -1469,6 +1469,1361 @@
     "type": "RECORD",
     "descriptions": "Data Derived from Crossref Events"
   },
+  {
```

Comment:
This doi table schema file is quite out of date and not actually used yet.

I feel like it would be better to modify this PR: https://github.com/The-Academic-Observatory/academic-observatory-workflows/pull/164 so that it is able to pull in the schemas for the various data sources (PubMed, OpenAlex, Unpaywall etc) that are used in the DOI table, rather than manually copy each one.

Would you be keen to figure out how to do that?

Reply:
Yeah sure. I'll have a look into it.